lib/fireblocks/fireblocks.go

```diff
@@ -0,0 +1,16 @@
+package fireblocks
+
+import (
+	"context"
+
+	f "github.com/omni-network/omni/lib/fireblocks/api"
```

Comment:
as mentioned in the past, avoid these type of aliased imports, this isn't our style.